CONTRIBUTING.md

```diff
@@ -7,6 +7,26 @@ The purpose of this document is to provide simple guidelines to help make your c
 
 There are many types of issues you can take on when contributing to the project "AgentCloud". We try our best to provide a wide array of open issues that vary in levels of complexity. From beginners to seasoned developers, everyone should be able to find something to work on.
 
+## Developing AgentCloud
+
+- [How Can I Contribute?](#how-can-i-contribute)
+- [First let's find the perfect open issue for you.](#first-lets-find-the-perfect-open-issue-for-you)
+  - [Join the community](#join-the-community)
+- [🚨 Contributing best practices](#-contributing-best-practices)
+- [How to contribute? 👨‍💻](#how-to-contribute-)
+  - [Issue assignment](#issue-assignment)
+  - [Fork and Pull Request Flow ⏳](#fork-and-pull-request-flow-)
+- [Commit Message Format 💬](#commit-message-format-)
+- [Keeping your Fork Up-to-Date 🆕](#keeping-your-fork-up-to-date-)
+- [Opening PRs 📩](#opening-prs-)
+- [Reviewing Pull Requests 🕵️](#reviewing-pull-requests-)
+- [Reporting Bugs 🐛](#reporting-bugs-)
+- [Suggesting Enhancements 🗣️](#suggesting-enhancements-)
```

Comment:
"Reviewing Pull Requests" and "Suggesting Enhancements" links don't work for me, i.e they don't scroll to the correct section.

Reply:
I can see all links are working. 
![image](https://github.com/rnadigital/agentcloud/assets/16003228/8d128b73-326f-47de-97ea-b94dea3acdbd)